examples/gocean/eg4/README

```diff
@@ -0,0 +1,79 @@
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+# FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+# COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+# INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+# BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#------------------------------------------------------------------------------
+# Author A. R. Porter, STFC Daresbury Lab
+
+The directory containing this file contains an example of the use of
+PSyclone to transform kernels that access variables and routines
+via module use statements.
+
+In order to use PSyclone you must first install it, ideally with pip.
+See ../../../README.md for more details.
+
+PSyclone can be run in the directory containing this file by 
+executing, e.g.::
+
+    psyclone -api "gocean1.0" alg.f90
+
+This will produce 'vanilla' PSy code along with a re-written version of
+the algorithm.
+
+The example algorithm file (alg.f90) contains a single Invoke that
+calls three kernels. Each of these illustrates a different aspect
+of kernels accessing data/routines via module use statements:
+
+1. kern_use_var - an example of a kernel that uses a variable
+   declared in another module;
+
+2. kern_call_kern - a kernel that calls another routine (kernel) defined
+   in another module;
+
+3. kern_nested_use - a kernel that calls another routine that itself
+   uses data defined in another module.
+
+These various forms of kernel only present a problem if a user wishes
+to transform them e.g. for use in either an OpenACC or OpenCL
+application. For instance, although the following:
+
+    psyclone -api "gocean1.0" -s ./acc_transform.py alg.f90
+
+produces PSy code with suitable OpenACC directives *and* transformed
+versions of the kernels (with `!$acc routine` directives added), it
+is *not* a valid OpenACC application. This is because the data_mod::gravity
+variable read by `kern_use_var` is not available on the remote device.
+Issue #323 will solve this problem by converting any module data
```

Comment:
Strictly speaking, an issue reports and/or discusses a problem and a pr addresses/fixes a problem - or at least in my mind this is the case.

Reply:
That is a bit picky but I can see what you're saying so I've reworded.